README.md

```diff
@@ -270,6 +270,8 @@ You are ready to deploy:
 $ git push heroku master
 ```
 
+EmberCLI compilation happens at deploy-time, triggered by the `asset:precompilation` rake task.
```

Comment:
I copy-pasted `EmberCLI` and `precompilation` from the Capistrano instructions to try to keep things consistent. Should I change them too?

Somewhere more prominent would be nice, especially if it's a general feature of the gem and not only something that happens on Heroku. It seems like a really nice architectural feature / benefit and worth telling people about.